libs/ui/inputs/src/lib/dropdown-selector/dropdown-selector.component.html

```diff
@@ -9,7 +9,9 @@
   <select
     [id]="id"
     (change)="this.selectValue.emit($event.target.value)"
-    class="text-main bg-white border border-gray-300 py-[10px] px-[16px] rounded-lg leading-tight focus:outline-none focus:border-primary"
+    class="text-main bg-white border border-gray-300 py-[10px] px-[16px] rounded-lg leading-tight focus:outline-none focus:border-primary {{
+      extraClass
```

Comment:
[[ngClass](https://angular.io/api/common/NgClass)] might be cleaner
not a big deal